edb/lang/schema/_std.eql

```diff
@@ -205,6 +205,26 @@ CREATE FUNCTION std::current_datetime() -> std::datetime
 CREATE FUNCTION std::uuid_generate_v1mc() -> std::uuid
     FROM SQL 'SELECT edgedb.uuid_generate_v1mc()';
 
+# JSON functions
+CREATE FUNCTION std::json_typeof($json: std::json) -> std::str
+    FROM SQL FUNCTION 'jsonb_typeof';
+
+CREATE FUNCTION std::json_array_as_array($arr: std::json) -> array<std::json>
```

Comment:
Drop this in favor of upcoming support for json casting (`<array<json>>json_array_value`)